### PR TITLE
[QPPA-2734] Creating a new QCDR measures script for 2019

### PR DESCRIPTION
**Pull request by chouinar**
https://jira.cms.gov/browse/QPPA-2734

Script to update QCDR measures for 2019. Heavily based on 2018 script. Has been tested, but the 2019 data isn't 100% finalized yet (format is, some values are still not correct in input). Additionally the package version needs to be updated as well still.

* [ ] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [ ] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

### CODE REVIEW

File: scripts/measures/2019/import-qcdr-measures.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    if (proportion === TRUE_CSV && continuous === FALSE_CSV && ratio === FALSE_CSV) {
+      // returns an integer if passed string '3', NaN if passed 'N/A'
+      const numPerformanceRates = _.parseInt(_.trim(record[referencedFields.numberOfPerformanceRates]));
+      if (_.isInteger(numPerformanceRates) && numPerformanceRates > 1) {
+        addMultiPerformanceRateDetails(newMeasure, record, qcdrStrataNamesDataPath, config);
+      } else {
+        newMeasure['metricType'] = 'registrySinglePerformanceRate';
+      }
+    } else {
+      newMeasure['metricType'] = 'nonProportional';
```
Comment (lamroger) [inline scripts/measures/2019/import-qcdr-measures.js]: Want to double check this is desired

Comment (lamroger) [reply scripts/measures/2019/import-qcdr-measures.js]: It's not, i reverted back to nonProportion

**Comment by kyeah** [comment]
👋 is there anything else that needs to be updated here besides re-running to fix `nonProportional`?

**Comment by lamroger** [comment]
@kyeah I think this is the last of the changes. See the last three commits. Could you double check?

**Comment by lamroger** [comment]
I ran `npm run build:measures 2019` and `cat measures/2019/measures-data.json | node scripts/validate-data.js measures 2019`

**Comment by lamroger** [comment]
Forgot to bump version but looks like https://github.com/CMSgov/qpp-measures-data/pull/265 will bump the version

**Pull request merged**